### PR TITLE
MBM-46 added myrial locker

### PR DESCRIPTION
**Pull request by aat79**

### CODE REVIEW

**Comment by MattKirchh** [comment]
The `myrial_locker.json` in models/block needs the same treatment that `myrial_steel_block.json` got. Replace the `minecraft:` with `minebound:` and it needs a `myrial_locker.json` in models/item. Looks like this got missed in review.

**Review by MattKirchh — APPROVED** [review]

File: src/main/resources/assets/minebound/blockstates/myrial_locker.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "variants": {
+    "": {
+      "model": "minebound:block/myrial_locker"
```
Comment (insertStringHere) [inline src/main/resources/assets/minebound/blockstates/myrial_locker.json:4]: this needs facing variants; it's not a uniform block

**Review by insertStringHere — CHANGES_REQUESTED** [review]

**Review by insertStringHere — COMMENTED** [review]
Made updates to directional blocks

**Review by insertStringHere — DISMISSED** [review]

**Review by MattKirchh — DISMISSED** [review]

File: src/main/resources/assets/minebound/models/block/myrial_locker.json [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
-    "side": "minecraft:block/myrial_locker_side"
+    "particle": "minebound:block/myrial_locker_side",
+    "down": "minebound:block/myrial_locker_side",
+    "up": "minebound:block/myrial_locker_side",
+    "north": "minebound:block/myrial_locker_front",
+    "east": "minebound:block/myrial_locker_side",
+    "south": "minebound:block/myrial_locker_side",
+    "west": "minebound:block/myrial_locker_side"
   }
 }
```
Comment (MattKirchh) [inline src/main/resources/assets/minebound/models/block/myrial_locker.json:12]: How come you reverted Mason's changes here? `orientable` seems to work.

**Review by MattKirchh — APPROVED** [review]

**Review by solidwaterslayer — APPROVED** [review]

**Pull request merged**